apps/events-helsinki/src/domain/search/eventSearch/constants.tsx

```diff
@@ -96,28 +96,28 @@ export enum EVENT_SEARCH_FILTERS {
 export const CATEGORY_CATALOG = {
   [EventTypeId.Course]: {
     default: [
-      COURSE_CATEGORIES.MOVIE,
-      COURSE_CATEGORIES.LANGUAGES,
-      COURSE_CATEGORIES.LITERATURE,
-      COURSE_CATEGORIES.ARTS_AND_CULTURE,
-      COURSE_CATEGORIES.VISUAL_ARTS,
-      COURSE_CATEGORIES.HANDICRAFTS,
-      COURSE_CATEGORIES.SPORT,
-      COURSE_CATEGORIES.MUSIC,
-      COURSE_CATEGORIES.GAMES,
-      COURSE_CATEGORIES.FOOD,
-      COURSE_CATEGORIES.DANCE,
-      COURSE_CATEGORIES.THEATRE,
+      EVENT_CATEGORIES.MOVIE,
+      EVENT_CATEGORIES.LANGUAGES,
+      EVENT_CATEGORIES.LITERATURE,
+      EVENT_CATEGORIES.ARTS_AND_CULTURE,
+      EVENT_CATEGORIES.VISUAL_ARTS,
+      EVENT_CATEGORIES.HANDICRAFTS,
+      EVENT_CATEGORIES.SPORT,
+      EVENT_CATEGORIES.MUSIC,
+      EVENT_CATEGORIES.GAMES,
+      EVENT_CATEGORIES.FOOD,
+      EVENT_CATEGORIES.DANCE,
+      EVENT_CATEGORIES.THEATRE,
     ],
     landingPage: [
-      COURSE_CATEGORIES.LITERATURE,
-      COURSE_CATEGORIES.VISUAL_ARTS,
-      COURSE_CATEGORIES.HANDICRAFTS,
-      COURSE_CATEGORIES.SPORT,
-      COURSE_CATEGORIES.MUSIC,
-      COURSE_CATEGORIES.GAMES,
-      COURSE_CATEGORIES.DANCE,
-      COURSE_CATEGORIES.THEATRE,
+      EVENT_CATEGORIES.LITERATURE,
+      EVENT_CATEGORIES.VISUAL_ARTS,
+      EVENT_CATEGORIES.HANDICRAFTS,
+      EVENT_CATEGORIES.SPORT,
+      EVENT_CATEGORIES.MUSIC,
+      EVENT_CATEGORIES.GAMES,
+      EVENT_CATEGORIES.DANCE,
+      EVENT_CATEGORIES.THEATRE,
```

Comment:
This does not look right when compared to the Events v. 2.0 where there were both the Hobbies and the Events combined in the same app: https://github.com/City-of-Helsinki/events-helsinki-ui/blob/latest-with-courses/src/domain/eventSearch/constants.tsx

The mapping for both the event types exists in that same file and they are not the same! I'm in assumption that we should have that mapping used that we had in version 2.0.

![image](https://user-images.githubusercontent.com/389204/201615239-6c2fe77b-ebd4-42be-972b-e5382faf512a.png)

![image](https://user-images.githubusercontent.com/389204/201615158-8154d1a7-8988-42f1-80e3-f0a0da950fee.png)

Reply:
Might be that I'm wrong and the mapping was ok, but just wanted to raise a concern about it.